packages/service/test/cli/gen-schema.test.mjs

```diff
@@ -1,9 +1,22 @@
-import { test } from 'tap'
+import { test, beforeEach, afterEach } from 'tap'
 import fs from 'fs/promises'
+import { mkdtempSync } from 'fs'
+import { tmpdir } from 'os'
 import { generateJsonSchemaConfig } from '../../lib/gen-schema.js'
+import { join } from 'path'
+
+let tmpDir
+beforeEach(async () => {
+  tmpDir = await mkdtempSync(join(tmpdir(), 'test-create-platformatic-'))
+})
+
+afterEach(() => {
+  fs.rmdir(tmpDir, { recursive: true, force: true })
+  process.env = {}
+})
 
 test('generateJsonSchemaConfig generates the file', async (t) => {
-  process.chdir('./test/tmp')
```

Comment:
Not sure why we use these folders here instead of one in `temp`, I suspect this causes flakiness in windows. Changed.